web/src/todo_widget.ts

```diff
@@ -199,15 +230,15 @@ export function activate(opts) {
                 return;
             }
             const key = $(e.target).attr("data-key");
-
-            const data = task_data.handle.strike.outbound(key);
+            const data = task_data.handle.strike.outbound(key ?? "");
             callback(data);
         });
     }
 
-    $elem.handle_events = function (events) {
+    $elem.handle_events = function (events): void {
         for (const event of events) {
-            task_data.handle_event(event.sender_id, event.data);
+            // eslint-disable-next-line @typescript-eslint/consistent-type-assertions
```

Comment:
Here, the types for JQuery handle_events function is declared globally in `poll_widget.ts`. 
This makes it difficult to pass the data as argument to the function as the types `HandleTaskData` and `InboundData` kind of overlap with each other and I could not get the type inferred conditionally.

Would love some guidance on what can I do here... Or should I leave it be?